features/salt.feature

```diff
@@ -15,3 +15,25 @@ Feature: Check if SaltStack Master is configured and running
   Scenario: Check SaltStack Master is properly configured
     When I issue command "ss -nta | grep 9080"
     Then it should contain "127.0.0.1:9080" text
```

Comment:
I am not sure how a mortal would relate that "ss -nta | grep 9080" to the feature.

Especially if I go to the [Salt Documentation](http://docs.saltstack.com/en/latest/topics/tutorials/firewall.html) and I read that salt master listens on ports 4505 and 4506. 9080, by my own reasoning is the port you chose for the API. So this scenario is a) confusing b) incomplete.

I suggest this reads like documentation:

```cucumber
Scenario: Check SaltStack Master is properly configured
   Then the Salt rest-api should be listening on local port 9080
   And the salt-master should be listening on public port 4505
  And the salt-master should be listening on public port 4506     
```
Then in the *details* (the steps) you can translate the public/9080 to a cal to ss looking for  127.0.0.1:9080 and the call to public 4506 to ss greping for *:4506.

It is not different to the problem we had in other scenarios that leak the DOM stuff to the Scenario descriptions.